release-team/role-handbooks/communications/README.md

```diff
@@ -118,72 +166,113 @@ This is a general example of what a release cycle might look like and the order
+        <li>If needed, solicit author(s) for a 'Deprecations and Removals' blog and get a placeholder PR in k/website for tracking
+        </ul>
+        </td>
+    </tr>
+    <tr>
+        <td>8</td>
+        <td></td>
+        <td>
+        <ul>
+        <li>Post reminders for the feature blog opt-in on the SIG slack channels (<a href="https://kubernetes.slack.com/archives/C0BP8PW9G/p1622556913118800">for example</a>)
+        <li>Assign feature blog topics as they come in to team shadows for support and tracking efforts
+        <li>Join the `#sig-docs-blog` channel on Slack and attend the bi-weekly meetings. Share the current status of the Feature blog opt-in's and work with the team to establish review expectations and publication strategy.
         </ul>
         </td>
     </tr>
     <tr>
-        <td>6</td>
+        <td>10</td>
         <td></td>
         <td>
         <ul>
+        <li>Send out final reminders for feature blog opt-in on the SIG slack channels
+        <li>Facilitate or start writing the optional Deprecations and Removals blog
+        </ul>
+        </td>
+    </tr>
+    <tr>
+        <td>11</td>
+        <td><b>Feature blog freeze</b></td>
+        <td>
+        <ul>
+        <li>Start attending burndown meetings
+        <li>Participate in the release retro part 1
+        <li>Feature blog freeze is this week
+        <li>Assign remaining feature blog topics
+        <li>Request placeholder PRs in k/website from all feature blog authors
+        <li>Update `#sig-docs-blog` with final tracked feature blogs and establish a regular cadence status check-in with the team there
+        </ul>
+        </td>
+    </tr>
+    <tr>
+        <td>12</td>
+        <td></td>
+        <td>
+        <ul>
+        <li>Begin the release blog draft, if not yet started
+        <li>Optional 'Deprecations and Removals' blog ready for review
+        <li>Schedule the release Live Webinar with CNCF via <a href="https://calendly.com/cncfonlineprograms">Calendly</a>, typically scheduled 3-4 weeks after the release
         <li>Schedule press and analyst pre-briefings and interviews for the release lead with CNCF
         <li>Schedule release blog and press embargo with CNCF
         </ul>
         </td>
     </tr>
     <tr>
-        <td>7</td>
+        <td>13</td>
         <td><b>Code Freeze</b></td>
         <td>
         <ul>
-        <li>Send release blog draft to CNCF
+        <li>Finalize and publish the 'Deprecations and Removals' blog once code freeze is in place.
+        <li>Update release blog draft, post-Code Freeze
+        <li>Check status on all feature blog PRs. Keep `#sig-docs-blog` up-to-date for editorial review, and establish tech reviewers are available 
+        <li>Check with Release Notes lead on content for Major Themes and Known Issues sections of the release blog
+        <li>Ensure that short one-to-two paragraph summaries of each Major Theme are available for the release blog from Release Notes lead or SIG Chairs
```

Comment:
```suggestion
```
Let's move this a week after Code Freeze. When Code Freeze starts, Release Notes will solicit Major Themes

Reply:
👍